packages/fluentui/react-northstar/test/utils/sharedIsConformant.tsx

```diff
@@ -0,0 +1,15 @@
+import { isConformant, IsConformantOptions } from '@fluentui/react-conformance';
+import { mountWithProvider as mount } from 'test/utils';
+import { Ref, RefFindNode } from '@fluentui/react-component-ref';
+// import { FocusZone } from '@fluentui/react-bindings';
+
+export function sharedIsConformant(testInfo: IsConformantOptions) {
+  const { disabledTests = [], wrapperComponents = [] } = testInfo;
+
+  isConformant({
+    ...testInfo,
```

Comment:
This won't handle merging disabledTests or wrapperComponents arrays properly. In this package you can use `_.merge` from lodash.

Also, you should put the defaults first and then merge in the custom options, so the custom options take precedence. `_.merge(/*default options*/, testInfo)`